src/Entity/GoldenBook.php

```diff
@@ -29,6 +30,7 @@ class GoldenBook
 
     /**
      * @ORM\Column(type="string", length=400)
+     * @Assert\Regex("/^\w+/")
```

Comment:
Do you really want to have only alphanumerical (and _) characters ?
No punctuation ?